integration/gadget-integration-tests-job.yaml

```diff
@@ -0,0 +1,41 @@
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: gadget-test
+  labels:
+    name: gadget-test
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: gadget-test
+  namespace: gadget-test
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: gadget-test
+subjects:
+- kind: ServiceAccount
+  name: gadget-test
+  namespace: gadget-test
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: gadget-test
+  namespace: gadget-test
+spec:
+  template:
+    spec:
+      serviceAccount: gadget-test
+      containers:
+      - name: gadget-test
+        image: docker.io/kinvolk/gadgettest:alban-integration_test3-b38f330
```

Comment:
Any idea how to keep this updated automatically? Maybe this should be latest here and the github action should replace it by the correct tag?

Reply:
sounds good, I'll do that